src/applications/disability-benefits/526EZ/components/ConfirmationPoll.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import Raven from 'raven-js';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
+
+import get from '../../../../platform/utilities/data/get';
+import { apiRequest } from '../../../../platform/utilities/api';
+
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+export const submissionStatuses = {
+  // Statuses returned by the API
+  pending: 'submitted', // Submitted to EVSS, waiting response
+  retry: 'retrying',
+  succeeded: 'received', // Submitted to EVSS, received response
+  exhausted: 'exhausted', // EVSS is down or something; ran out of retries
+  failed: 'non_retryable_error', // EVSS responded with some error
+  // When the api serves a failure
+  apiFailure: 'apiFailure'
+};
+
+const terminalStatuses = [
+  submissionStatuses.succeeded,
+  submissionStatuses.exhausted,
+  submissionStatuses.retry,
+  submissionStatuses.failed
+];
+
+const successMessage = (claimId) => (
+  <div>
+    <p>Thank you for filing a claim for increased disability compensation.</p>
+    <strong>Claim ID number</strong>
+    <div>{claimId}</div>
+    <p>You can check the status of your claim online. Please allow 24 hours for your increased disability claim to show up there.</p>
+    <p><a href="/track-claims">Check the status of your claim.</a></p>
+    <p>If you don’t see your increased disability claim online after 24 hours, please call Veterans Benefits Assistance at <a href="tel:+18008271000">1-800-827-1000</a>, Monday – Friday, 8:00 a.m. – 9:00 a.m. (ET).</p>
+  </div>
+);
+
+const checkLaterMessage = (jobId) => (
+  <div>
+    <p>Thank you for filing a claim for increased disability compensation.</p>
+    <strong>Confirmation number</strong>
+    <div>{jobId}</div>
+    <p>You can check the status of your claim online. Please allow 24 hours for your increased disability claim to show up there.</p>
+    <p><a href="/track-claims">Check the status of your claim.</a></p>
+    <p>If you don’t see your increased disability claim online after 24 hours, please call Vets.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a>, Monday – Friday, 8:00 a.m. – 9:00 a.m. (ET).</p>
+  </div>
+);
+
+const errorMessage = (jobId) => (
+  <div>
+    <p>We’re sorry. Something went wrong on our end when we tried to submit your application. For help, please call the Vets.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a>, Monday – Friday, 8:00 a.m. – 9:00 a.m. (ET).</p>
+    <strong>Confirmation number</strong>
+    <div>{jobId}</div>
+    <br/>
+  </div>
+);
+
+// TODO: Make this a loading spinner
+const pendingMessage = (longWait) => {
```

Comment:
I mean, I guess these messages could be in `helpers` or something just to clean this component up a bit, but I absolutely don't have a strong opinion since it's not like they're used anywhere else.

Reply:
Yeah, I was thinking about moving them to a content file. That can happen outside of this PR, though (for sake of time).